frontend/src/components/Table.tsx

```diff
@@ -22,9 +23,11 @@ type TableProps<DataType> = {
   toolbarContent?: React.ReactElement<typeof ToolbarItem>;
   emptyTableView?: React.ReactElement<typeof Tr>;
   caption?: string;
-  disableRowRenderSupport?: boolean;
-  tbodyProps?: TbodyProps & { ref?: React.Ref<HTMLTableSectionElement> };
-} & Omit<TableComposableProps, 'ref' | 'data'>;
+} & EitherNotBoth<
+  { disableRowRenderSupport?: boolean },
+  { tbodyProps?: TbodyProps & { ref?: React.Ref<HTMLTableSectionElement> } }
```

Comment:
If you wanted to -- you could make this EitherOrNone -- and make them required. Just realized that by looking at it now... sorry about that.